addons/binding/org.openhab.binding.knx/ESH-INF/thing/ip.xml

```diff
@@ -0,0 +1,268 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="knx"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- Bridge Thing Type -->
+    <bridge-type id="ip">
+        <label>KNX/IP gateway</label>
+        <description>This Bridge is for IP based KNX gateways</description>
+        
+        <channels>
+            <channel id="errorsall" typeId="errors" />
+            <channel id="errors5min" typeId="errors" />
+        </channels>
+        
+        <config-description>
+            <parameter name="ipAddress" type="text">
+                <label>Network Address</label>
+                <description>Network address of the KNX/IP gateway</description>
+                <required>true</required>
+            </parameter>
+            <parameter name="portNumber" type="integer">
+                <label>Port Number</label>
+                <description>Port number of the KNX/IP gateway</description>
+                <required>true</required>
+            </parameter>
+            <parameter name="localIp" type="text">
+                <label>Local Network Address</label>
+                <description>Network address of the local host to be used to set up the connection to the KNX/IP gateway</description>
+                <required>false</required>
+            </parameter>      
+            <parameter name="ipConnectionType" type="text">
+                <label>IP Connection Type</label>
+                <description>The ip connection type for connecting to the KNX bus. Could be either TUNNEL or ROUTER</description>
+                <required>true</required>
```

Comment:
I'd also use one setting as default (TUNNEL?) and you should provide the two options through an `<options>` element (then you can shorten the description again)